trade-finance-manager-ui/templates/case/facility/_macros/facility_details.njk

```diff
@@ -100,7 +100,13 @@
 
       <dl class="govuk-grid-row ukef-list ukef-list-one-column">
         <dt class="govuk-grid-column-one-quarter govuk-body-s">Inclusion notice received</dt>
-        <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-inclusion-notice-received">{{ facility.dates.inclusionNoticeReceived | localiseTimestamp("d MMMM yyyy", user.timezone) | dashIfEmpty }}</dd>
+        <dd class="govuk-grid-column-three-quarters govuk-body-s" data-cy="facility-inclusion-notice-received">
+          {% if facility.facilityStage === 'Commitment' %}
```

Comment:
I am not sure why we are comparing facility stage, I would have expected to see submission type rather as if the application's submission type is `MIA` then `-` otherwise the inclusion notice timestamp.